src/GuiBase/Timeline/HelpDialog.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef RADIUMENGINE_TIMELINE_HELP_HPP__
+#define RADIUMENGINE_TIMELINE_HELP_HPP__
+
+#include <QDialog>
+
+namespace Ui {
+class HelpDialog;
+} // namespace Ui
+
+namespace Ra::GuiBase {
+
+/// Dialog to display navigation/interaction controls.
```

Comment:
Well, not so clear as reference documentation. (related to comment about mouse/keyboard interactions in this review)